NEWS.md

```diff
@@ -1,3 +1,42 @@
+flux-core version 0.20.0 - 2020-09-30
+-------------------------------------
+
+This release features changes to support Flux as the native resource
+manager on small (<= 256 node) clusters, for testing only.  A draft system
+administration guide is available at:
+
+https://flux-framework.readthedocs.io/en/latest/adminguide.html
+
+### New features
+
+ * hwloc: add printing of num GPUs to `flux hwloc info` (#3217)
+ * resource: mark nodes down when they are stopped (#3207)
+ * broker:  allow late-joining brokers, execute rc1/rc3 on all ranks (#3168)
+ * shell/pmi: add improved PMI key exchange mechanism (#3219)
+
+### Fixes
+
+ * job-info: handle annotations race (#3196, #3208)
```

Comment:
PR #3196 is different from #3208 perhaps could be

job-manager / job-info: synchronize job priority changes from job-manager to job-info (#3208)

Reply:
Oh thanks, I messed that up.

I broke out 3208 again and scrunched your description down just a bit to fit on a line:
> * job-manager: communicate job priority changes manager to job-info (#3208)

Hope that's OK

Reply:
Did you mean "from manager to job-info?" If that's longer than a line,  maybe just "to job-info".

Reply:
DOH!